src/Microsoft.PowerShell.Commands.Utility/commands/utility/UtilityCommon.cs

```diff
@@ -238,16 +238,19 @@ public override string ToString()
                     // Display each byte, in 2-digit hexadecimal, and add that to the left-hand side.
                     nextLine.AppendFormat("{0:X2} ", currentByte);
 
-                    // If the character is printable, add its ascii representation to
-                    // the right-hand side.  Otherwise, add a dot to the right hand side.
-                    if ((currentByte >= 0x20) && (currentByte <= 0xFE))
-                    {
-                        asciiEnd.Append((char)currentByte);
-                    }
-                    else
-                    {
-                        asciiEnd.Append('.');
+                    // Add its ascii representation to the right-hand side.
+                    // If the byte represents a control character, replace with a space for byte value 0   
+                    // and the Unicode replacement character ('/uFFFD') for all others.
+                    char charToAppend = (char)currentByte;
+                    if (char.IsControl(charToAppend)) {
+                        // replace all 0 value bytes with spaces for better visualisation of any surrounding ascii text
+                        if (currentByte == 0) {
+                            charToAppend = ' ';      
+                        } else {
+                            charToAppend = '\uFFFD';      // Standard Unicode replacement character for all other control chars
```

Comment:
Please remove obvious comment.

Reply:
I disagree.  I don't think people should have to lookup what character this is.

Reply:
The 'for all other control chars` part is not correct 
```suggestion
                            charToAppend = '\uFFFD';      // Standard Unicode replacement character
```

Reply:
Ok.
The comment should be on separate line (CodeFactor request).